pymedphys/_mosaiq/constants.py

```diff
@@ -35,3 +35,16 @@
     20: "User Defined",
     21: "PDR",
 }
+
+ORIENTATION = {
+    0: "Unknown",
+    1: "HFS",
+    2: "HFP",
+    3: "HFRL",
+    4: "HFRR",
+    5: "FFS",
+    6: "FFP",
+    7: "FFRL",
+    8: "FFRR",
+    9: "Sitting",
```

Comment:
Do you know if these are the same for everyone?

Reply:
These should be universal, however I did change the format for them. I changed them to line up with DICOM, but as @sjswerdloff mentioned below, some of them are off. Here is a screenshot from the Mosaiq Data Dictionary where the values were defined.

![image](https://user-images.githubusercontent.com/40006991/89440380-27c72580-d711-11ea-98be-22008b0803b6.png)

Reply:
Probably best not a good idea to screenshot that pdf, if it's the one I think it is.

Reply:
Oops, I wont make that mistake again 🤦‍♂️

Reply:
All good :).